content/blog/2020-pycon-india-mentorship.md

```diff
@@ -0,0 +1,43 @@
+Title: Writing a proposal for the first time? Mentorship team is here to help.
+Date: 2020-07-28 12:34 
+Author: Ankita Prakash (@Anpr1211)
+Category: 2020 
+Slug: announce-pycon-india-mentorship 
+
+Announcing PyCon India Mentorship team's resources, contact links, and weekly calls for first-time speakers as well as veterans.
+
+<!-- PELICAN_END_SUMMARY -->
+
+> Do you have that one idea that can make the world “a better place”? (Silicon Valley Fans, Assemble!)
+
+Do you wish to submit a proposal to PyCon India 2020 but cannot figure out the flow of your talk or how to present it?
+
+We know it's not always easy to find the right words when you have something to share. It happens to the best of us. But no need to worry, as the PyCon India mentorship team is here to help you out with your proposal. Whether you are submitting a proposal at PyCon India for the first time or you are a veteran speaker with years of experience behind you, our mentorship team will guide you through the process and help you wherever required in a number of ways as mentioned below. We want to help you put your best foot forward when it comes to submitting proposals! 
+ 
+## Recommended Resources from the Content Team 
+
+Prospective speakers of PyCon India can take a look at our curated blog posts on writing better talk and workshop proposals.  
+- [Tips on writing a “kickass” proposal](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html) 
+- [Guidelines on creating video previews](https://in.pycon.org/blog/2020/2020-call-for-proposals-announcement.html)
+
+## Join us on Zulip for faster answers to any questions
+
+Moreover, our mentors will be in constant touch on Zulip. You can join them on the [PyCon India Zulip Channel](https://pyconindia.zulipchat.com/#) and drop a text in the [pre-cfp topic](https://pyconindia.zulipchat.com/#narrow/stream/245487-wg.2Fmentorship/topic/pre-cfp) of the #Mentorship stream (Streams are available to choose from the sidebar on the left). 
+
+## Open Mentorship calls 
+
+In the hopes of providing better opportunities and assistance for first-time speakers at PyCon India 2020, we are starting weekly mentorship calls. This would be an open call for participants and potential speakers to interact with PyCon India mentors to review, share, and discuss draft proposals. Details of the call are as follows:
+
+Date: Weekly on Saturdays 
+Time: 15:00 to 17:00 IST
+Platform: Google Meet - Join with the [invite link](https://meet.google.com/pot-jzjq-udn)
```

Comment:
Same here

Reply:
I added another line break between each of the lines. So it does seem better and is presented in new lines.